### PR TITLE
Create Terraform vars for test/dev cluster, CP4D, etc 

### PR DESCRIPTION
**Pull request by aishwaryabk**
Fixes: #184 
Signed-off-by: Cloud User <Aishwarya.Kamat@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **prajyot-parab** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @prajyot-parab` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
Welcome @aishwaryabk! It looks like this is your first PR to ocp-power-automation/ocp4-upi-powervs 🎉

**Comment by ltccci** [comment]
Hi @aishwaryabk. Thanks for your PR.

I'm waiting for a [ocp-power-automation](https://github.com/orgs/ocp-power-automation/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/ocp-power-automation/ocp4-upi-powervs#contributing) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: compute-vars/large.tfvars
Comment (bpradipt) [inline compute-vars/large.tfvars:7]: we can make some modifications to the large
1. processors = "1.5" for all types of nodes
2. Memory = 64 GB for all OCP nodes
3. Worker count = 3

Comment (aishwaryabk) [reply compute-vars/large.tfvars:7]: Updated the large template.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-582171965" title="LGTM">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: compute-vars/large.tfvars
Comment (bpradipt) [inline compute-vars/large.tfvars:4]: Sorry a miss from me. The bootstrap should be really   {memory      = "32",   processors  = "0.5",  "count"   = 1}

File: compute-vars/large.tfvars
Comment (bpradipt) [inline compute-vars/large.tfvars:6]: Also make worker=4

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

File: compute-vars/large.tfvars
Comment (aishwaryabk) [reply compute-vars/large.tfvars]: Updated these changes.

File: compute-vars/large.tfvars
Comment (aishwaryabk) [reply compute-vars/large.tfvars]: Updated the file.

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-585323236" title="LGTM">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: compute-vars/small.tfvars
Comment (yussufsh) [inline compute-vars/small.tfvars:5]: Only worry is that 16GB masters may fail the deployment with OOM issues. So it is of no use to give this flavor at all. We have experienced this on Power systems. @bpradipt @arcprabh

Comment (arcprabh) [reply compute-vars/small.tfvars:5]: That is right @yussufsh . Please set 32GB for the rhcos nodes for now.
For 4.8 onwards with RHEL 8.4, if the OOM issue is not recreated with 16GB, then we can consider updating it to 16.

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

File: compute-vars/small.tfvars
Comment (aishwaryabk) [reply compute-vars/small.tfvars]: Updated the file with the changes

File: compute-vars/medium.tfvars
Comment (bpradipt) [inline compute-vars/medium.tfvars:6]: To differentiate small from medium flavor lets have 3 worker nodes. 
I realise we will never be able to arrive at a consensus on good values for small/med/large. So this could be an acceptable compromise

Comment (aishwaryabk) [reply compute-vars/medium.tfvars]: Changed worker count to 3.

**Review by bpradipt — DISMISSED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-592081925" title="LGTM">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yussufsh — CHANGES_REQUESTED** [review]
@aishwaryabk  please refer this commit to #184 and give a good commit message.

**Comment by ltccci** [comment]
New changes are detected. LGTM label has been removed.

**Comment by aishwaryabk** [comment]
> @aishwaryabk please refer this commit to #184 and give a good commit message.

Changes have been updated.

**Review by yussufsh — APPROVED** [review]

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-592081925" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-592894862" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#" title="Author self-approved">aishwaryabk</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-592895377" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/216#pullrequestreview-592894862" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
